plugins/inputs/mavlink/sample.conf

```diff
@@ -0,0 +1,37 @@
+# Read metrics from a Mavlink connection to a flight controller.
+[[inputs.mavlink]]
+     # Flight controller URL. Must be a valid Mavlink connection string in one
```

Comment:
Comments need to have double escaped: `##`

Reply:
Fixed.